src/MessagePack.Experimental/UnsafeBlittableFormatter/UnsafeUnmanagedStructArrayFormatter.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+#pragma warning disable SA1649 // File name should match first type name
+
+namespace MessagePack.Formatters
+{
+    public sealed unsafe class UnsafeUnmanagedStructArrayFormatter<T> : IMessagePackFormatter<T[]?>
+        where T : unmanaged
+    {
+        public const sbyte ExtensionTypeCode = 51;
```

Comment:
Is this a new extension type code? If so, we should probably add it to `ThisLibraryExtensionTypeCodes`.